app/src/main/java/ingsw/group1/findmyphone/event/LogEventType.java

```diff
@@ -0,0 +1,93 @@
+package ingsw.group1.findmyphone.event;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.util.ArrayMap;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.util.Map;
+
+import ingsw.group1.findmyphone.R;
+
+/**
+ * Enum defining the possible kind of Events the Log is supposed to keep track of.
+ *
+ * @author Riccardo De Zen
+ */
+public enum LogEventType {
```

Comment:
Doesn't extend `EnumTyped`?

Reply:
It's more like the opposite, an EnumTyped class is specified on an Enum